src/ess/livedata/dashboard/plot_orchestrator.py

```diff
@@ -607,43 +775,83 @@ def parse_raw_cell(self, cell_data: dict[str, Any]) -> PlotCell | None:
         Parse a raw cell dict into a typed PlotCell.
 
         Use this to convert cells from templates or persisted configurations
-        into typed objects that can be passed to :py:meth:`add_grid`.
+        into typed objects that can be passed to :py:meth:`add_cell`.
+
+        Supports two formats:
+        - Legacy format: 'geometry' + 'config' (single layer)
+        - New format: 'geometry' + 'layers' (multiple layers)
 
         Parameters
         ----------
         cell_data
-            Cell configuration dict with 'geometry' and 'config' keys.
-            The config must contain: workflow_id, source_names, plot_name.
+            Cell configuration dict with 'geometry' and either 'config' or 'layers'.
+            Each layer/config must contain: workflow_id, source_names, plot_name.
             Optional: output_name, params.
 
         Returns
         -------
         :
-            Parsed PlotCell, or None if the plotter is unknown (cell skipped).
+            Parsed PlotCell, or None if all plotters are unknown (cell skipped).
         """
-        config_data = cell_data['config']
-        plot_name = config_data['plot_name']
-
-        # Validate params, skipping cells with unknown plotters
-        params = self._validate_params(plot_name, config_data.get('params', {}))
-        if params is None:
-            return None
-
         geometry = CellGeometry(
             row=cell_data['geometry']['row'],
             col=cell_data['geometry']['col'],
             row_span=cell_data['geometry']['row_span'],
             col_span=cell_data['geometry']['col_span'],
         )
+
+        # Handle both legacy 'config' and new 'layers' format
+        if 'layers' in cell_data:
+            raw_layers = cell_data['layers']
+        elif 'config' in cell_data:
+            # Legacy format: wrap single config in list
+            raw_layers = [cell_data['config']]
+        else:
+            self._logger.warning('Cell has neither config nor layers, skipping')
+            return None
+
+        layers: list[Layer] = []
+        for layer_data in raw_layers:
+            layer = self._parse_raw_layer(layer_data)
+            if layer is not None:
+                layers.append(layer)
+
+        if not layers:
+            return None
+
+        return PlotCell(geometry=geometry, layers=layers)
+
+    def _parse_raw_layer(self, layer_data: dict[str, Any]) -> Layer | None:
```

Comment:
Do the changes to the config require updates to the `src/ess/livedata/config/instruments/bifrost/grid_templates/*.yaml` files?

Also, are the changes here retro-compatible or do they not work with old grid config files?

Reply:
Note the legacy-code that should allow reading old config files. But yes, we will want to update our files, latest when we remove legacy handling. Once files are converted they cannot be read with an old dashboard though.